docs/Getting-Started.md

```diff
@@ -131,59 +131,91 @@ npm install --save fastify-plugin
 
 **server.js**
 ```js
-const fastify = require('fastify')({
-  logger: true
-})
+'use strict'
 
-fastify.register(require('./our-db-connector'), {
-  url: 'mongodb://localhost:27017/'
-})
-fastify.register(require('./our-first-route'))
+const fastify = require('fastify')({logger: true})
+const database = require('./database')
+const routes = require('./routes')
 
-fastify.listen(3000, function (err, address) {
-  if (err) {
-    fastify.log.error(err)
-    process.exit(1)
-  }
-  fastify.log.info(`server listening on ${address}`)
-})
+const start = async () => {
+    fastify.register(database)
+    fastify.register(routes)
+
+    try {
+        await fastify.listen(3000, '0.0.0.0')
+    } catch (err) {
+        fastify.log.error(err)
+        process.exit(1)
+    }
+}
+
+start()
 ```
 
-**our-db-connector.js**
+**database.js**
 ```js
+'use strict'
+
+/* 
+- In a terminal start up mongodb
+
+- create the test database
+> use test_database
+
+- create collection
+> db.createCollection('test_collection')
+
+- insert a row
+db.test_collection.insert({name: 'John Smith'})
+
+- show the newly inserted row and copy the _id string
+> db.test_collection.find().pretty()
+
+- use the _id string in the url example localhost:3000/search/5ef583456f8921125bb83b20
+*/
+
 const fastifyPlugin = require('fastify-plugin')
-const MongoClient = require('mongodb').MongoClient
 
-async function dbConnector (fastify, options) {
-  const url = options.url
-  delete options.url
+async function dbConnector(fastify, options) {
+    const { ObjectId, MongoClient } = require('mongodb')
```

Comment:
I think we should use `fastify-mongodb` instead. I know this also teaches about our plugin system, but I think it's too much info for a "getting started".

Reply:
Ok i'll give it another try using fastify-mongodb. Thank you for taking a moment to look this over!